plugin/kubernetes/README.md

```diff
@@ -109,6 +110,10 @@ kubernetes [ZONES...] {
 * `ignore empty_service` return NXDOMAIN for services without any ready endpoint addresses (e.g., ready pods).
   This allows the querying pod to continue searching for the service in the search path.
   The search path could, for example, include another Kubernetes cluster.
+* `external` will synthesize records in the listed **[ZONES...]** for all Kubernetes services with Load Balancer IPs.
```

Comment:
which ZONES is this referencing?

Reply:
The local one in the option.  I'll clarify...